pxtcompiler/emitter/transpile.ts

```diff
@@ -76,9 +76,10 @@ namespace ts.pxtc.transpile {
         }
```

Comment:
please add globalNames here if you wouldn't mind
I think that might solve some other naming issues